cmd/kubectl_token.go

```diff
@@ -135,28 +135,32 @@ func CredentialCommand() cli.Command {
 }
 
 func runCredential(ctx *cli.Context) error {
-	if ctx.Bool("delete") {
-		return deleteCachedCredential(ctx)
-	}
```

Comment:
Seems like it might've been a flag in the past, which is no longer true. We use the `delete` subcommand for this.

Reply:
One of the reason because we should try to avoid passing the `*cli.Context`, and have a `Config` struct to pass. But that's a refactor for another day.

Reply:
IMHO passing the context or not is a different story.

The issue here is that it seems like that delete was a flag 
`rancher token --delete <name>`
and then turned into an argument (subcommand)
`rancher token delete <name>`